GooglePlacesAutocomplete.js

```diff
@@ -553,6 +557,41 @@ const GooglePlacesAutocomplete = React.createClass({
     this.setState({listViewDisplayed: true});
   },
 
+  _shouldShowPoweredLogo() {
+
+     if (this.state.dataSource.getRowCount() == 0) {
+       return false
+     }
+
+     for (let i=0;i<this.state.dataSource.getRowCount();i++) {
+        let row  = this.state.dataSource.getRowData(0, i);
```

Comment:
The indentation is random. Please format the changes to use 2-space indentation.